Tests/MusicXMLTests/ScoreTests/README.md

```diff
@@ -0,0 +1,28 @@
+# ScoreTests
+
+## Test Case Structure
+The score tests consist of test suite executor and test cases. Any individual test case is a folder that satisfy the following rules.
+
+- The folder name is arbitrary
+- The folder must contain a `manifest.plist` file, otherwise the folder is not treated as a test case and thus skipped.
+
+### Manifest
+The `manifest.plist` further defines the test case.
+- The manifest is a dictionary at its root.
+- The manifest MUST contain a `DisplayName` String field to specify the test name.
+- The manifest MUST contain an `XMLFileName` String field to specify the test file name. The file name includes the file name and the extension. An uncompressed score file with that name MUST exists in the same folder as the manifest.
+- The manifest MAY contain an `ExpectationClassName` String field to specify the Swift class name that contains the expectation of the parsed MusicXML. The expectation class is elaborated in the next section. If not provided, equal check will not be conducted.
+- The manifest MAY contain an `Description` String field for a textual description of the score.
+- The manifest MAY contain an `Source` String field for an URL link to the source.
+
+
+### Expectation Class
+- The expectation MUST be a `class`. This is a restriction since `struct` is not dynamically lodable.
```

Comment:
I think this could be enforced by constraining the `protocol` to `AnyObject` as mentioned elsewhere.

Reply:
lodable -> loadable